languagetool-language-modules/de/src/main/resources/org/languagetool/resource/de/hunspell/spelling.txt

```diff
@@ -55067,4 +55067,6 @@ Caffè Latte
 Caffè mocha
 Café au Lait
 Café Cortado
-Galão
\ No newline at end of file
+Galão
+shipped #eng
+Shipping #eng
```

Comment:
wollen wir englische wörter aufnehmen?

Reply:
Außerdem: ist die eingedeutschte Variante für 'shipped' nicht 'geshippt' (Präteritum analog)? Eventuell könnten wir diese Formen auch aufnehmen, wenn wir schon dabei sind

Reply:
ich finde immer so denglische Ausdrücke können user in ihr eigenes wörterbuch aufnehmen, oder?

Reply:
Da sind schon 1268 englische Begriffe in der spelling.txt, also ja, aber natürlich nur bei häufiger im Deutschen vorkommenden englischen Begriffen...

Reply:
@St-ac-y , ich weiß nicht. Wir haben zwei Regeln zu falscher Partizipbildung für Anglizismen. Wenn diese Regeln etwas vorschlagen, das dann als Rechtschreibfehler markiert wird, ist das nicht so gut. Inwiefern 'shippen' abgedeckt ist oder sein sollte, weiß ich nicht.

Reply:
dann kann man wörter wie shippt etc. auch noch einbauen. bin kein fan davon, aber wenn wir so regeln haben, dann ist es blöd, wenn ein loop entsteht.

Reply:
wenn aufnehmen, dann vielleicht eher in die `ignore.txt` statt `spelling.txt`